### PR TITLE
Add CMU text to spirit.urdf and spirit_meshes.urdf

### PR DESCRIPTION
**Pull request by jcnorby**
Adds "Carnegie Mellon University" to text on the side of the robot in both polygon and mesh urdfs.

### CODE REVIEW

**Review by qishuny — APPROVED** [review]

**Comment by qishuny** [comment]
Looks cool

**Comment by qishuny** [comment]
Looks cool!

File: quad_simulator/spirit_description/urdf/spirit_meshes.urdf [hunk 1]
```diff
@@ -1020,5 +1023,23 @@
...
+      <geometry>
+        <mesh filename="package://spirit_description/meshes/Spirit40_CMU_Text_transformed.stl" scale="1.3 1 1.3"/>
+      </geometry>
+      <material name="scots-rose"/>
+    </visual>
+    <inertial>
+      <mass value="0.0001"/>
+      <inertia ixx="0.00" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
+    </inertial>
+  </link>
```
[the remark this answers is not in the document]
Comment (jcnorby) [reply quad_simulator/spirit_description/urdf/spirit_meshes.urdf:1038]: Fixed.

**Comment by jrenaf** [comment]
Great, looks good

**Review by jrenaf — APPROVED** [review]

**Pull request merged**